src/styles/blocks/image.scss

```diff
@@ -0,0 +1,10 @@
+.image {
+  padding: 32px 19px 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.image__display--settings {
+  width: 160px;
```

Comment:
Consider defining the width value as a variable in a separate file to adhere to the task requirements for maintainability.